jerry-ext/jerry-context/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
```

Comment:
Isn't the file name should be  jerry-ext/jerryx-context/CMakeLists.txt ? Or simply jerry-ext/context/CMakeLists.txt ?